_dev/gulp/tasks/js/wrap.js

```diff
@@ -39,6 +44,7 @@ module.exports = function (gulp, plugins, settings, handlers) {
             // Rename file.
             .pipe(rename(function (path) {
```

Comment:
Just to keep it consistent: You don't need to require ALL modules that will use in the code... Just those which are not prefixed with `gulp-`. Because we use `gulp-load-plugins` module. That module is in-charge to load gulp plugins for us, and we are already using it a lot in the project. All gulp plugins are already loaded, and passed in the function by `plugins` argument.

Here, you could use `plugins.rename()` instead `rename()`. And avoid line `var rename = require('gulp-rename');`